### PR TITLE
docs: add issue template

### LINKED ISSUE #159: Create Issue Creation Template
### Scope 
- Create an issue creation template to facilitate the creation of issues in the repository. 
- Add the template in the .github folder of this repository. 

### How to implement the template?
The template must have at least these parts:
- Scope
- How to implement the issue?
- How to apply?
Please follow the contribution guidelines in the [readme](https://github.com/ByteBuildersLabs/ByteBeastsFrontend/blob/main/README.md) file
- Feel free to add new points to the template

### How to apply?
Please follow the contribution guidelines in the [readme](https://github.com/ByteBuildersLabs/ByteBeastsFrontend/blob/main/README.md) file

### PR DESCRIPTION
**Pull request by Ibinola**
# Pull Request Overview

## 📝 Summary
Add Issue Template

### Related Issues
- Closes #159

### Type of Change
Mark with an `x` all the checkboxes that apply (like `[x]`).
- [x] 📝 Documentation (updates to README, docs, or comments)
- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] 👌 Enhancement (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)

### What's Changed
Added an issue template to the `_github` folder

### Implementation Details
- Created a template with clear sections

### Technical Notes
N/A

## 🔧 Tests Results
N/A

### Test Coverage
- [ ] Unit Tests
- [ ] Integration Tests
- [x] Manual Testing

### Evidence
N/A

### Testing Notes
N/A

## 🔜 Next Steps
N/A

## Summary by CodeRabbit

- **New Features**
	- Introduced a structured issue template for GitHub to standardize issue reporting.
	- The template includes sections for title, scope, implementation steps, acceptance criteria, additional context, contribution guidelines, and a checklist.

### CODE REVIEW

**Comment by Ibinola** [comment]
Please review @jimenezz22

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### 📸 Additional Context
+
+[Add any additional information, screenshots, or relevant details that might help in understanding the issue]
+
+### Contribution Guidelines
+
+- Please ensure you read and follow the contribution guidelines in the project's [README](https://github.com/ByteBuildersLabs/ByteBeastsFrontend/blob/main/README.md) file
+
+### ✅ Checklist
```
Comment (jimenezz22) [inline .github/ISSUE_TEMPLATE.md]: can you delete this please ?The checklist part

**Review by jimenezz22 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Ibinola) [reply .github/ISSUE_TEMPLATE.md]: just to confirm, only checklist section or everything from line 30 down ?

Comment (jimenezz22) [reply .github/ISSUE_TEMPLATE.md]: everything from line 32 down

**Review by jimenezz22 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Ibinola) [reply .github/ISSUE_TEMPLATE.md]: fixed

**Review by jimenezz22 — APPROVED** [review]
lgtm

**Pull request merged**